libfwupdplugin/fu-plugin.h

```diff
@@ -52,6 +52,38 @@ typedef enum {
 	FU_PLUGIN_VERIFY_FLAG_LAST
 } FuPluginVerifyFlags;
 
+/**
+ * FuPluginInternalFlags:
+ *
+ * The plugin internal flags, not exported over D-Bus.
+ **/
+typedef enum {
+	/**
+	 * FU_PLUGIN_INTERNAL_FLAG_NONE:
+	 *
+	 * No flags set.
+	 *
+	 * Since: 1.9.14
+	 */
+	FU_PLUGIN_INTERNAL_FLAG_NONE = 0,
+	/**
+	 * FU_PLUGIN_INTERNAL_FLAG_DEFAULT_DEVICE_GTYPE:
+	 *
+	 * Use the first-added plugin GType when there are multiple to choose from.
```

Comment:
I think it's better to explicitly specify which one is the default instead of rely upon a first added behvaior.  Otherwise we're a very nuanced bug away from compiler optimization bugs or unintentionally changing the order in a refactor.

Reply:
Good point. I guess a few more options:

 * We use fu_device_set_priority() and then choose the device with the biggest (or lowest) priority -- this isn't ideal as it's also used to decide the plugin priority.
 * We add something like FU_DEVICE_INTERNAL_FLAG_DEFAULT_GTYPE although it's not clear if two gtypes have that set.
 * We add something like fu_plugin_set_device_gtype_default() and just store `GType device_gtype_default` in the `FuPluginPrivate`

I guess easiest is the 3rd option.

Reply:
Me also: https://github.com/fwupd/fwupd/pull/6846